GenderPayGap.Database.Models/Organisation.cs

```diff
@@ -36,8 +36,11 @@ public Organisation()
         public DateTime? LastCheckedAgainstCompaniesHouse { get; set; }
         public bool OptedOutFromCompaniesHouseUpdate { get; set; } = false;
 
+        [Obsolete]
         public string SecurityCode { get; set; }
+        [Obsolete]
         public DateTime? SecurityCodeExpiryDateTime { get; set; }
+        [Obsolete]
         public DateTime? SecurityCodeCreatedDateTime { get; set; }

```

Comment:
Why are these marked as obsolete rather than being deleted?

Reply:
I'm planning to remove them (and create database migration) in a separate PR